src/storage/scripting.cc

```diff
@@ -409,13 +390,23 @@ Status FunctionCall(redis::Connection *conn, const std::string &name, const std:
     std::string libcode;
     s = srv->FunctionGetCode(libname, &libcode);
     if (!s) return s;
-
     s = FunctionLoad(conn, libcode, false, false, &libname, read_only);
     if (!s) return s;
 
     lua_getglobal(lua, (REDIS_LUA_REGISTER_FUNC_PREFIX + name).c_str());
   }
 
+  ScriptFlags function_flags = read_only ? ScriptFlagType::kScriptNoWrites : 0;
+  lua_getglobal(lua, (REDIS_LUA_REGISTER_FUNC_FLAGS_PREFIX + name).c_str());
+  if (lua_isstring(lua, -1)) {
+    function_flags |= GET_OR_RET(ParseInt<ScriptFlags>(lua_tostring(lua, -1)));
+  }
```

Comment:
As I mentioned before, I think we can just store the flags (as an integer) in `RedisRegisterFunction` to `REDIS_LUA_REGISTER_FUNC_FLAGS_PREFIX + name`, instead of parsing it every time in `FunctionCall`.

Reply:
Lua does not support direct storage of uint64_t and requires static_cast, so I chose to save string

Reply:
The string here is the string form of uint64_t (parsing result). The parsing process has been completed during Function Load instead of in Function Call.

Reply:
If there is any deviation in my understanding, please point it out, thanks.

Reply:
hmm I think maybe signed integer is fine here?

Reply:
Yes, I think using signed integer is also an option. Actually, I tried using signed integer at first, but a strange feeling prompted me to switch to string. I suddenly felt uncomfortable looking at static_cast. 

This is just my very subjective feeling, HaHa. If you mind, I will change it to store as integers later.